WordPress/Classes/ViewRelated/NewGutenberg/NewGutenbergViewController.swift

```diff
@@ -137,13 +128,8 @@ class NewGutenbergViewController: UIViewController, PostEditor, PublishingEditor
         self.editorSession = PostEditorAnalyticsSession(editor: .gutenbergKit, post: post)
         self.navigationBarManager = navigationBarManager ?? PostEditorNavigationBarManager()
 
-        var conf = EditorConfiguration(blog: post.blog)
-        conf.title = post.postTitle ?? ""
-        conf.content = post.content ?? ""
-        conf.postID = post.postID?.intValue != -1 ? post.postID?.intValue : nil
-        conf.postType = post is Page ? "page" : "post"
```

Comment:
It appears we no longer set the post ID or type. I was unable to locate a breakage from this, but we should likely reinstate this.